composer.json

```diff
@@ -19,7 +19,7 @@
   "require": {
     "gorriecoe/silverstripe-link": "^1.0",
```

Comment:
Just testing now, it looks OK.  Once merged this is going to need to be a 2.0 release, since SS5 is technically a breaking change.  More of a note for myself, thanks for this! :+1:

Reply:
I don't think it needs to be a new major version since there are no functional changes here, it's just allowing this module to work with CMS5. Minor should be fine imho.